### PR TITLE
Bug/fix communication bug between react&firebase for post #69

### LINKED ISSUE #69: Bug: Web application cannot post pictures to firebase (Story Point: 1)
Currently, the request to post a picture to firebase isnt working. Communication between the react form to post picture and the handler function on firebase isn't working.

The handler function is working properly. Check code in pull request #50 . 

Based off #17.

### PR DESCRIPTION
**Pull request by Eltom777**
Fixed #69

### CODE REVIEW

File: picturepoint/src/AddPhoto/AddPhoto.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import axios from "axios";
```
Comment (Eltom777) [inline picturepoint/src/AddPhoto/AddPhoto.jsx:2]: I rebased to kill the conflict but I don't know what just happened. Good news though, the conflict is gone and src/AddPhoto/AddPhoto.jsx was updated

Comment (Eltom777) [reply picturepoint/src/AddPhoto/AddPhoto.jsx:2]: actually, nevermind they are still there :(

Comment (Eltom777) [reply picturepoint/src/AddPhoto/AddPhoto.jsx:2]: Now it's fixed

**Review by a-iacampo — APPROVED** [review]
approved

**Pull request merged**